awx/main/models/inventory.py

```diff
@@ -960,7 +960,10 @@ def ad_hoc_commands(self):
     RelatedJobsMixin
     '''
     def _get_related_jobs(self):
-        return InventoryUpdate.objects.filter(inventory_source__in=self.inventory_sources.all())
+        return UnifiedJob.objects.non_polymorphic().filter(
```

Comment:
These `OR` Q filters can be extremely heavy... how does this affect the query times?

Reply:
This was stolen from the method of the same name for the `Inventory` model

https://github.com/AlanCoding/awx-1/blob/36fb8755868a369499f1afc966c2e1d7d02d217b/awx/main/models/inventory.py#L496-L504

I see the argument that groups are deleted much more often than inventory, but this is kind of needed for the concept of our active job deletion protection.